templates/mailprog.sh.epp

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# mailprog.sh: Managed by Puppet
+
+# SCRIPT ARGS SENT BY SLURM:
+# first arg is '-s'
+# last arg is recipient email address
+recipient=${@: -1}
+# all middle args are the subject
+subject=${@:2:$#-2}
+
+# send email with empty body and appropriate subject to the recipient
```

Comment:
Why is the body empty? Does slurm do some magic to get the body in there?

Reply:
Slurm doesn't seem to send a body to MailProg (I don't recall seeing one, I checked some messages I've received, and looked at the code for v23.11.4 and it doesn't look like it ever would). I clarified that in the template. Thanks!